libfreerdp/crypto/test/TestFuzzCryptoErReadUniversalTag.c

```diff
@@ -0,0 +1,10 @@
+#include <freerdp/crypto/crypto.h>
+
+extern int LLVMFuzzerTestOneInput(const uint8_t* Data, size_t Size)
+{
+	bool tag = true;
+	bool pc = true;
+	er_read_universal_tag((wStream*)Data, tag, pc);
```

Comment:
this is really wrong, you can not simply cast any buffer to a `wStream` and expect it to not crash

Reply:
Done.